packages/client/src/components/AppNavBar.tsx

```diff
@@ -1,11 +1,62 @@
 import MenuIcon from '@mui/icons-material/Menu';
 import ShoppingBasketIcon from '@mui/icons-material/ShoppingBasketOutlined';
-import { AppBar, Box, Button, Fab, IconButton, Link, Menu, MenuItem, Toolbar, useTheme } from '@mui/material';
+import {
+  AppBar,
+  Box,
+  Button,
+  Fab,
+  IconButton,
+  Link,
+  Menu,
+  MenuItem,
+  Palette,
+  Theme,
+  Toolbar,
+  useTheme,
+} from '@mui/material';
 import { useState } from 'react';
 
 import logo from '../assets/logo.svg';
 
-const PAGES = ['O nas', 'Kontakt', 'Dostawa'] as const;
+const PAGES = [
+  {
+    name: 'O nas',
+    href: 'about',
+    pos: 'left',
+  },
+  {
+    name: 'Kontakt',
+    href: 'contact',
+    pos: 'left',
+  },
+  {
+    name: 'Dostawa',
+    href: 'delivery',
+    pos: 'left',
+  },
+  {
+    name: 'Logowanie',
+    href: 'login',
+    pos: 'right',
+    color: (palette: Palette) => palette.primary.main,
+  },
+  {
+    name: 'Rejestracja',
+    href: 'register',
+    pos: 'right',
+    color: (palette: Palette) => palette.secondary.dark,
+  },
+] as const;
+
+const pageToButton = (page: typeof PAGES[number], theme: Theme) => (
+  <Button
+    key={page.href}
+    href={page.href}
+    sx={{ color: 'color' in page ? page.color(theme.palette) : theme.palette.secondary.contrastText, px: 2 }}
```

Comment:
Nie musisz przekazywać theme możesz użyć [callback](https://mui.com/system/the-sx-prop/#callback-values), wtedy uprości się też `map(pageToButton)`.